src/components/FeaturedProducts.jsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import featuredProducts from '../assets/data/featured-products';
```

Comment:
Same comment as above. Move this to the parent component and pass the data as props.